arches_arcgispro_addin/CreateResource.xaml.cs

```diff
@@ -53,12 +40,15 @@ public static async Task<List<GeometryNode>> GetGeometryNode()
 
                 response.EnsureSuccessStatusCode();
                 string responseBody = await response.Content.ReadAsStringAsync();
-                var serializer = new JavaScriptSerializer();
-                dynamic results = serializer.Deserialize<dynamic>(@responseBody);
+                dynamic results = JsonConvert.DeserializeObject<dynamic>(@responseBody);
 
                 foreach (dynamic element in results)
                 {
-                    nodeidResponse.Add(new GeometryNode(element["resourcemodelname"], element["name"], element["nodeid"]));
+                    var name = (string)element["name"];
+                    if (name == "Geospatial Coordinates")
```

Comment:
This does not work for the geojson node with the different name.
Instead maybe we can something like this.

```
var resourceName = (string)element["resourcemodelname"];
if (resourceName != "Arches System Settings")
```

Reply:
Thanks @njkim. Funnily enough, I was actually going to do it that way first of all. Will make suggested change and also include the compiled version too.

Reply:
All done.